### PR TITLE
Python certifi win32

### PR DESCRIPTION
**Pull request by y2kbugger**
Used Skeleton for pypi as starting point
Added license_file and maintainer, and urls
Added host dep of setuptools_scm

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by y2kbugger** [comment]
@conda-forge-admin, please ping team
@conda-forge/help-python

I am only the maintainer of this recipe, not of the upstream project.

File: recipes/python-certifi-win32/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - certifi_win32
+
+about:
+  home: "https://gitlab.com/alelec/python-certifi-win32"
+  license: BSD
+  license_family: BSD
+  license_file: "./LICENSE"
```
Comment (xhochy) [inline recipes/python-certifi-win32/meta.yaml]:
```suggestion
  license_file: "LICENSE"
```

File: recipes/python-certifi-win32/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - wincertstore
+    - wrapt >=1.10.4
+
+test:
+  imports:
+    - certifi_win32
+
+about:
+  home: "https://gitlab.com/alelec/python-certifi-win32"
+  license: BSD
```
Comment (xhochy) [inline recipes/python-certifi-win32/meta.yaml]: Which BSD license?

File: recipes/python-certifi-win32/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: e074663a1b6e54233f28d15832f9872440472144abafa432ec7f5d473fa33085
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - certifi
```
Comment (xhochy) [inline recipes/python-certifi-win32/meta.yaml]: You probably only need pip, setuptools_scm and Python in `host`

File: recipes/python-certifi-win32/meta.yaml (hunk 2 above)
Comment (y2kbugger) [reply recipes/python-certifi-win32/meta.yaml]: Seems to be: `BSD 2-clause "Simplified" License` Do I put that whole phrase in the yaml file?

Comment (y2kbugger) [reply recipes/python-certifi-win32/meta.yaml]: I've added it.

File: recipes/python-certifi-win32/meta.yaml (hunk 3 above)
Comment (y2kbugger) [reply recipes/python-certifi-win32/meta.yaml]: Fixed https://github.com/conda-forge/staged-recipes/pull/10576/commits/a3b5f4feefa708c4720bd50cf2cb3c4b94e5a5cb

File: recipes/python-certifi-win32/meta.yaml (hunk 2 above)
Comment (y2kbugger) [reply recipes/python-certifi-win32/meta.yaml]: See https://github.com/conda-forge/staged-recipes/pull/10576/commits/a3b5f4feefa708c4720bd50cf2cb3c4b94e5a5cb

**Comment by y2kbugger** [comment]
@conda-forge-admin, please ping team

Looks like I might be fully ready now.

File: recipes/python-certifi-win32/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - wincertstore
+    - wrapt >=1.10.4
+
+test:
+  imports:
+    - certifi_win32
+
+about:
+  home: "https://gitlab.com/alelec/python-certifi-win32"
+  license: "BSD 2-clause Simplified"
```
Comment (xhochy) [inline recipes/python-certifi-win32/meta.yaml]: We use https://spdx.org/licenses/ for license names
```suggestion
  license: BSD-2-Clause
```

**Comment by y2kbugger** [comment]
Given this is for Windows only, am I skipping the other platforms correctly? I skip them in the recipe but that they're still runs for them is that normal?

**Comment by xhochy** [comment]
> Given this is for Windows only, am I skipping the other platforms correctly? I skip them in the recipe but that they're still runs for them is that normal?

Yes, the jobs start but then see that there is no work and then finish quickly.

**Pull request merged**